apps/accounts/views.py

```diff
@@ -155,9 +156,10 @@ class AuthorizeView(APIView):
     def get(self, request, user, identity, next_path):
         if user is None:
             register_user(request, identity)
+            return HttpResponseRedirect(reverse('users.edit'))
         else:
             login_user(request, user, identity)
-        return HttpResponseRedirect(next_path or '/')
+            return HttpResponseRedirect(next_path or '/')
```

Comment:
since I always seem to need them when debugging, I like to have log statements in if branches like this. However, I don't have a clear debugging use case in mind so we could always add them later.